changelogs/fragments/8517-cmd-runner-lang-auto.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - CmdRunner module utils - the parameter ``force_lang`` now supports the special value ``auto`` which will automatically try and determine the best parsable locale in the system (https://github.com/ansible-collections/community.general/pull/8517).
```

Comment:
Maybe also mention the `puppet` module (and update it's documentation), since it passes its `environment_lang` parameter straight on to `force_lang`.

Reply:
I recently found out about this function in core's module util, so I thought of that as a feature here. I have not tested that in other systems/modules, etc... I would rather work on that in separate PRs.

Reply:
Eventually, I think we should make `auto` the default value for the parameter, rather than `C`, but that requires deprecation periods, and a certain amount of work in testing.

Reply:
Hmm, but this PR already directly modifies the `puppet` module, so at least some minimal indication should be added there? Or let's wait until today's release is out, then you can also split it up into multiple PRs :)

Reply:
Not sure I agree with the wording  "modifies the `puppet` module". It opens a new possibility for users of that module, sure, but there is modification in the code.
As I mentioned I have not tested it thoroughly yet, so not very keen on advertising it. If it works, it might even prompt the removal of that option from `puppet`.

Reply:
@felixfontein as I see it, `puppet` would need only an update in docs - which usually does not grant a changelog fragment. I have just updated that doc, but I still do not see this as a changelog fragment. Please advise.

Reply:
That's totally fine for me!